community-containers/jellyseerr/jellyseerr.json

```
@@ -12,12 +12,14 @@
                 {
                     "ip_binding": "%APACHE_IP_BINDING%",
                     "port_number": "5055",
-                    "protocol": "tcp"
+                    "protocol": "tcp",
+                    "host_port": "5055"
```

host_port is not supported by AIO. The `port_number` is already exposing the mentioned port on the host.

```suggestion
                    "protocol": "tcp"
```